tasks/78_valid-braces/valid-braces.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable max-len */
+//  First I wanted to use three different counters and loop with switch statement inside,
+//  to match number of opening and closing braces. And it worked, I passed all the tests but for one
```

Comment:
there is also a way to solve this task with stack structure [here you can find algorithm for this](https://www.geeksforgeeks.org/check-for-balanced-parentheses-in-an-expression/) for stack it is possible to use just an array